### PR TITLE
python.pkgs.flask_migrate: 1.7.0 -> 2.0.3

### PR DESCRIPTION
**Pull request by risicle**
Fixes build against new dependencies after some test tinkering

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/flask-migrate/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "107x78lkqsnbg92dld3dkagg07jvchp3ib3y0sivc4ipz6n1y7rk";
+  };
+
+  # When tests run with python3*, tests should run commands as "python3 <args>",
+  # not "python <args>"
+  patchPhase = ''
```
Comment (FRidh) [inline pkgs/development/python-modules/flask-migrate/default.nix]: postPatch, try not to override the default phases when its not needed

File: pkgs/development/python-modules/flask-migrate/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    substituteInPlace tests/test_migrate.py --replace "python" "${python.executable}"
```
Comment (FRidh) [inline pkgs/development/python-modules/flask-migrate/default.nix]: there's a symlink from `python` to `python3` so it should work fine without this fix

Comment (risicle) [inline pkgs/development/python-modules/flask-migrate/default.nix]: This wasn't added by me. I'll try removing it.

Comment (FRidh) [inline pkgs/development/python-modules/flask-migrate/default.nix]: it likely existed then before I added the symlink

Comment (risicle) [inline pkgs/development/python-modules/flask-migrate/default.nix]: Hmm. Seems happy. Will push that version.

**Pull request merged**